pulsar-broker/src/main/java/org/apache/pulsar/broker/service/AbstractSubscription.java

```diff
@@ -18,14 +18,37 @@
  */
 package org.apache.pulsar.broker.service;
 
+import org.apache.pulsar.common.api.proto.CommandSubscribe.IsolationLevel;
+
+import java.util.Map;
 import java.util.Optional;
 import java.util.concurrent.atomic.LongAdder;
 import java.util.function.ToLongFunction;
 
 public abstract class AbstractSubscription implements Subscription {
+    protected static final String SUBSCRIPTION_ISOLATION_LEVEL_PROPERTY = "pulsar.subscription.isolation.level";
```

Comment:
Since it is a prop for transaction, we should make this prop name contains the keyword `transaction`

Reply:
And for all the system properties, please add `__` as a prefix.